Documentation/gettingstarted/encryption-ipsec.rst

```diff
@@ -156,20 +158,20 @@ Validate the Setup
 ==================
 
 Run a ``bash`` shell in one of the Cilium pods with
-``kubectl -n kube-system exec -ti <cilium pod> -- bash`` and execute the
-following commands:
+``kubectl -n kube-system exec -ti ds/cilium -- bash`` and execute the following
+commands:
```

Comment:
Would you mind doing this change in the wireguard guide as well?

Reply:
It's already `ds/cilium` in the [wireguard guide](https://github.com/kaworu/cilium/blob/pr/kaworu/docs/encryption-ipsec/Documentation/gettingstarted/encryption-wireguard.rst#validate-the-setup).